### DISCUSSION TITLE
Improve the text editor (Tasks, Notes...) 📝

### DISCUSSION THREAD

**Discussion by Bonapara** [twentyhq/twenty#3647] [Ideas]
Related to #825

**Comment by BeHanAir** [comment]
wow

**Comment by brendanlaschke** [comment]
https://dev.to/shiwaforce/notion-like-text-editor-with-ai-autocomplete-and-planetscale-database-in-nextjs-using-shadcnui-4236 

I think with this it shouldn't be too hard to add ai autocompletion ...

**Reply by Bonapara** [reply]
This would be awesome indeed 🪄
